app/src/main/res/layout/item_campaign_small.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:id="@+id/root"
+    android:layout_width="wrap_content"
+    android:layout_height="@dimen/campaign_small_height">
+
+    <FrameLayout
+        android:layout_width="match_parent"
+        android:id="@+id/image_container"
+        android:layout_height="@dimen/campaign_small_height">
+
+        <ImageView
+            android:id="@+id/image"
+            android:layout_width="match_parent"
+            android:layout_height="@dimen/campaign_small_height"
+            android:scaleType="centerCrop"
+            android:src="@drawable/placeholder"/>
+
+        <View
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:background="@drawable/darken_30"/>
+    </FrameLayout>
+
+    <org.dosomething.letsdothis.ui.views.typeface.CustomTextView
```

Comment:
FIXME: Both the title and call to action should be centered. Right now only the title is centered.